ui/src/components/list-items/QasListItems.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="qas-list-items shadow-primary">
+    <q-list bordered class="round rounded-borders" dense separator>
+      <q-item v-for="(item, index) in list" :key="index" v-ripple :clickable="!useIconRedirect" :to="redirect(item)">
+        <slot :index="index" :item="item" name="item">
+          <q-item-section>
+            <slot :index="index" :item="item" name="item-section-left" />
+          </q-item-section>
+
+          <q-item-section side>
+            <slot :index="index" :item="item" name="item-section-side">
+              <qas-btn flat round :to="getRedirectPaylod(item)">
+                <q-icon v-bind="icon" />
+              </qas-btn>
+            </slot>
+          </q-item-section>
+        </slot>
+      </q-item>
+    </q-list>
+  </div>
+</template>
+
+<script>
+import QasBtn from '../btn/QasBtn.vue'
+
+export default {
+  components: {
+    QasBtn
+  },
+
+  props: {
+    icon: {
+      type: Object,
+      default: () => ({
+        name: 'o_chevron_right',
+        color: 'primary'
+      })
+    },
+
+    list: {
+      type: Array[Object],
+      default: () => []
+    },
+
+    redirectKey: {
+      type: String,
+      default: 'uuid'
+    },
+
+    useIconRedirect: {
+      type: Boolean
+    },
+
+    to: {
+      type: Object,
+      default: () => ({})
+    }
+  },
+
+  methods: {
+    redirect (item) {
+      return this.useIconRedirect ? undefined : this.getRedirectPaylod(item)
```

Comment:
`return this.useIconRedirect && this.getRedirectPaylod(item)` não teria o mesmo efeito?

Reply:
Não, vai retornar um false caso não passe, e preciso que retorne undefined.